gnucash/import-export/qif-imp/assistant-qif-import.c

```diff
@@ -1800,9 +1800,30 @@ gnc_ui_qif_import_load_progress_start_cb (GtkButton * button,
 
     /* This step will fill 70% of the bar. */
     gnc_progress_dialog_push (wind->load_progress, 0.7);
-    load_return = scm_call_5 (qif_file_load,
+
+    {
+        gsize length;
+        gchar *contents;
+        gchar *utf8_string;
+        GError *gerror;
+
+        g_file_get_contents(path_to_load, &contents, &length, NULL);
+
+        utf8_string = g_convert (contents, length, "UTF-8",
+                                 qif_encoding, NULL, NULL, &gerror);
+
+        /* bwahaha. crash. */
```

Comment:
*Rude*

Reply:
This whole `{section}` will probably be moved into a static function.